fulp_modules/features/antagonists/monster_hunter/hunting_contract.dm

```diff
@@ -78,7 +78,7 @@
 			if(!objectives_completed || length(owner.rabbits) || used_up)
 				return
 			if(!is_station_level(loc.z))
-				to_chat(usr, span_warning("The pull of the ice moon isn't strong enough here...."))
+				to_chat(usr, span_warning("The pull of the ice moon isn't strong enough here..."))
```

Comment:
```suggestion
				to_chat(usr, span_warning("The pull of the Ice Moon isn't strong enough here..."))
```

Im pretty sure its capitalized in other parts of the game but it doesnt seem like its respected everywhere either